apps/wpcom-block-editor/src/wpcom/features/tracking.js

```diff
@@ -73,11 +127,28 @@ const trackBlockReplacement = ( originalBlockIds, blocks ) => {
 	} );
 };
 
+/**
+ * Track inner blocks replacement.
+ * This is how Page Templates insert their content into page, by replacing everything that was already there.
+ *
+ * @param {Array} rootClientId id of parent block
+ * @param {object|Array} blocks block instance object or an array of such objects
+ * @returns {void}
+ */
+const trackInnerBlocksReplacement = ( rootClientId, blocks ) => {
+	trackBlocks( blocks, 'wpcom_block_inserted', block => ( {
+		block_name: block.name,
+		blocks_replaced: false,
```

Comment:
I think this should be true when doing a block replacement.

Reply:
does it mean that `blocks_replaced` should be true only for the root blocks? So for the inner blocks it should be false?

Reply:
If the newly inserted block replaced anything, it should be true. I think inner blocks should inherit it, as they are part of the replacement.